source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,239 @@
+.. _minio-certmanager-operator:
+
+=========================
+cert-manager for Operator
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+
+MinIO Operator manages TLS certificate issuing for the services hosted in the ``minio-operator`` namespace. 
+
+This page describes how to manage the Operator's TLS certificates with :ref:`cert-manager <minio-certmanager>`.
```

Comment:
This is more

```suggestion
This procedure assumes a new or fresh installation of the MinIO Operator configured to use cert-manager for TLS certificate management.
```

We should also consider how to document migrating an existing Operator config to cert-manager.

Reply:
Asking about existing installs internally. We can address that in a separate PR.
I modified your suggestion a bit, but did add that this is assuming a new/fresh install of Operator.

Reply:
Added a section on migrating an existing deployment.